phpstan.dist.neon

```diff
@@ -0,0 +1,6 @@
+parameters:
+    level: 1
+    paths:
+        - bin
```

Comment:
will phpstan actually check the bin files that have no extension ?

Reply:
Hmm, it does have a side-effect that it doesn't work on Windows... `vendor/bin/phpstan analyze bin/*` - the `*` is expanded by the shell, not PHP (and on Windows PHPStan simply complains that there is no "%PWD%/bin/*" file). :/